### PR TITLE
dnsx: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/dnsx.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# dnsx
+
+> Un equipo de herramientas DNS rápido y multipropósito para ejecutar múltiples consultas DNS.
+> Nota: la entrada a `dnsx` necesita ser pasada a través de `stdin` (pipe `|`) en algunos casos.
```
Comment (tricantivu) [inline pages.es/common/dnsx.md:4]:
```suggestion
> Nota: la entrada a `dnsx` necesita ser pasada a través de `stdin` (tubería `|`) en algunos casos.
```

File: pages.es/common/dnsx.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# dnsx
+
+> Un equipo de herramientas DNS rápido y multipropósito para ejecutar múltiples consultas DNS.
```
Comment (tricantivu) [inline pages.es/common/dnsx.md:3]:
```suggestion
> Un equipo de herramientas de DNS rápido y multipropósito para ejecutar múltiples consultas DNS.
```

File: pages.es/common/dnsx.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+> Nota: la entrada a `dnsx` necesita ser pasada a través de `stdin` (pipe `|`) en algunos casos.
+> Ver también: `dig`, `dog`, `dnstracer`.
+> Más información: <https://github.com/projectdiscovery/dnsx>.
+
+- Consulta el registro A de un (sub)dominio y muestra la [re]spuesta recibida:
```
Comment (tricantivu) [inline pages.es/common/dnsx.md:8]:
```suggestion
- Consulta el registro A de un subdominio y muestra la [re]spuesta recibida:
```

File: pages.es/common/dnsx.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+`echo {{ejemplo.com}} | dnsx -a -re`
+
+- Consulta todos los registros DNS (A,AAAA,CNAME,NS,TXT,SRV,PTR,MX,SOA,AXFR,CAA):
```
Comment (tricantivu) [inline pages.es/common/dnsx.md:12]:
```suggestion
- Consulta todos los registros DNS (A, AAAA, CNAME, NS, TXT, SRV, PTR, MX, SOA, AXFR y CAA):
```

File: pages.es/common/dnsx.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+`dnsx -recon -re <<< {{ejemplo.com}}`
```
Comment (tricantivu) [inline pages.es/common/dnsx.md:14]:
```suggestion
`dnsx -recon -re <<< {{ejemplo.com}}`
```

Not all shells support [_here-strings_ (i.e. `<<<`)](https://www.gnu.org/software/bash/manual/bash.html#Here-Strings). Maybe `echo {{ejemplo.com}} | dnsx -recon -re` would replace the current example.

Would it be worth documenting shell syntax for command examples in `common` to aim for portability @kbdharun @kant?

File: pages.es/common/dnsx.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+- Consulta un tipo específico de registro DNS:
+
+`echo {{ejemplo.com}} | dnsx -re -{{a|aaaa|cname|ns|txt|srv|ptr|mx|soa|any|axfr|caa}}`
+
+- Salida s[o]lo [r]espuesta (no muestra el dominio o subdominio consultado):
```
Comment (tricantivu) [inline pages.es/common/dnsx.md:20]:
```suggestion
- Muestra s[o]lo la [r]espuesta (no muestra el dominio o subdominio consultado):
```

File: pages.es/common/dnsx.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+`echo {{ejemplo.com}} | dnsx -ro`
+
+- Muestra la respuesta sin procesar de una consulta, especificando los solucionado[r]es a utilizar y los intentos de reintentos en caso de error:
+
+`echo {{ejemplo.com}} | dnsx -{{debug|raw}} -resolver {{1.1.1.1,8.8.8.8,...}} -retry {{número}}`
+
+- Fuerza bruta de registros DNS utilizando un marcador de posición:
+
+`dnsx -dominio {{FUZZ.ejemplo.com}} -wordlist {{ruta/a/lista_palabras.txt}} -re`
```
Comment (tricantivu) [inline pages.es/common/dnsx.md:30]:
```suggestion
`dnsx -dominio {{FUZZ.ejemplo.com}} -wordlist {{ruta/a/lista_de_palabras.txt}} -re`
```

File: pages.es/common/dnsx.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+- Consulta un tipo específico de registro DNS:
+
+`echo {{ejemplo.com}} | dnsx -re -{{a|aaaa|cname|ns|txt|srv|ptr|mx|soa|any|axfr|caa}}`
+
+- Salida s[o]lo [r]espuesta (no muestra el dominio o subdominio consultado):
+
+`echo {{ejemplo.com}} | dnsx -ro`
+
+- Muestra la respuesta sin procesar de una consulta, especificando los solucionado[r]es a utilizar y los intentos de reintentos en caso de error:
```
Comment (tricantivu) [inline pages.es/common/dnsx.md:24]:
```suggestion
- Muestra la respuesta sin procesar una consulta, especificando los solucionado[r]es a utilizar y el número de intentos en caso de haber errores:
```

File: pages.es/common/dnsx.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+`echo {{ejemplo.com}} | dnsx -{{debug|raw}} -resolver {{1.1.1.1,8.8.8.8,...}} -retry {{número}}`
+
+- Fuerza bruta de registros DNS utilizando un marcador de posición:
+
+`dnsx -dominio {{FUZZ.ejemplo.com}} -wordlist {{ruta/a/lista_palabras.txt}} -re`
+
+- Registros DNS de fuerza bruta a partir de una lista de [d]ominios y listas de palabras, adjuntando la salida [o] a un archivo sin códigos de [c]olor:
```
Comment (tricantivu) [inline pages.es/common/dnsx.md:32]:
```suggestion
- Aplica fuerza bruta a registros DNS a partir de una lista de [d]ominios y listas de palabras, adjuntando la salida a un archivo sin códigos de [c]olor:
```

File: pages.es/common/dnsx.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+`dnsx -dominio {{ruta/a/dominio.txt}} -wordlist {{ruta/a/lista_palabras.txt}} -re -output {{ruta/a/salida.txt}} -no-color`
+
+- Extrae registros `CNAME` desde la lista dada de subdominios, con una velocidad [l]imitante de consultas DNS por segundo:
```
Comment (tricantivu) [inline pages.es/common/dnsx.md:36]:
```suggestion
- Extrae registros `CNAME` desde una lista de subdominios, con una velocidad [l]ímite de consultas DNS por segundo:
```

File: pages.es/common/dnsx.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+- Salida s[o]lo [r]espuesta (no muestra el dominio o subdominio consultado):
+
+`echo {{ejemplo.com}} | dnsx -ro`
+
+- Muestra la respuesta sin procesar de una consulta, especificando los solucionado[r]es a utilizar y los intentos de reintentos en caso de error:
+
+`echo {{ejemplo.com}} | dnsx -{{debug|raw}} -resolver {{1.1.1.1,8.8.8.8,...}} -retry {{número}}`
+
+- Fuerza bruta de registros DNS utilizando un marcador de posición:
```
Comment (tricantivu) [inline pages.es/common/dnsx.md:28]:
```suggestion
- Aplica fuerza bruta a registros DNS utilizando un marcador de posición:
```

File: pages.es/common/dnsx.md (hunk 5 above)
Comment (kbdharun) [reply pages.es/common/dnsx.md:14]: > Would it be worth documenting shell syntax for command examples in `common` to aim for portability @kbdharun @kant?

Yeah

Comment (kant) [reply pages.es/common/dnsx.md:14]: Definitively yes!

File: pages.es/common/dnsx.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+
+`echo {{ejemplo.com}} | dnsx -{{debug|raw}} -resolver {{1.1.1.1,8.8.8.8,...}} -retry {{número}}`
+
+- Aplica fuerza bruta a registros DNS utilizando un marcador de posición:
+
+`dnsx -dominio {{FUZZ.ejemplo.com}} -wordlist {{ruta/a/lista_de_palabras.txt}} -re`
+
+- Aplica fuerza bruta a registros DNS a partir de una lista de [d]ominios y listas de palabras, adjuntando la salida a un archivo sin códigos de [c]olor:
+
+`dnsx -dominio {{ruta/a/dominio.txt}} -wordlist {{ruta/a/lista_palabras.txt}} -re -output {{ruta/a/salida.txt}} -no-color`
```
Comment (tricantivu) [inline pages.es/common/dnsx.md:34]:
```suggestion
`dnsx -dominio {{ruta/a/dominio.txt}} -wordlist {{ruta/a/lista_de_palabras.txt}} -re -output {{ruta/a/salida.txt}} -no-color`
```

**Review by tricantivu — APPROVED** [review]

**Pull request merged**